docs/reference/mapping/fields/ttl-field.asciidoc

```diff
@@ -104,3 +104,6 @@ may still be retrieved before they are purged.
 How many deletions are handled by a single <<docs-bulk,`bulk`>> request. The
 default value is `10000`.
 
+==== Note on `detect_noop`
+If an update doesn't change the `_source` document it's expiration time won't
```

Comment:
Maybe this should be more specific eg. "If an update tries to update the `_ttl` value but doesn't change the `_source` document"?

also s/it's/its/